tests/v2/validation/workloads/workload_test.go

```diff
@@ -316,38 +349,109 @@ func (w *WorkloadTestSuite) TestWorkloadPodScaleDown() {
 	replicas = int32(1)
 	scaleDownDeployment.Spec.Replicas = &replicas
 
+	log.Info("Updating deployment replicas")
 	scaleDownDeployment, err = deployment.UpdateDeployment(w.client, w.cluster.ID, namespace.Name, scaleDownDeployment)
 	require.NoError(w.T(), err)
 
 	w.validateDeploymentScale(namespace.Name, scaleDownDeployment, imageName, 1)
 }
 
+func (w *WorkloadTestSuite) TestWorkloadPauseOrchestration() {
+	subSession := w.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Creating new project and namespace")
+	_, namespace, err := projectsapi.CreateProjectAndNamespace(w.client, w.cluster.ID)
+	require.NoError(w.T(), err)
+
+	log.Info("Creating new deployment")
+	pauseDeployment, err := deployment.CreateDeployment(w.client, w.cluster.ID, namespace.Name, 2, "", "", false, false)
+	require.NoError(w.T(), err)
+
+	w.validateDeploymentScale(namespace.Name, pauseDeployment, imageName, 2)
+
+	log.Info("Pausing orchestration")
+	pauseDeployment.Spec.Paused = true
+	pauseDeployment, err = deployment.UpdateDeployment(w.client, w.cluster.ID, namespace.Name, pauseDeployment)
+	require.NoError(w.T(), err)
+
+	log.Info("Verifying orchestration is paused")
+	err = deployment.VerifyOrchestrationStatus(w.client, w.cluster.ID, namespace.Name, pauseDeployment.Name, true)
+	require.NoError(w.T(), err)
+
+	replicas := int32(2)
```

Comment:
Can we update the replicas to a different count? Setting the same number of replicas during an update as the existing config doesn't seem to add much value.